newscoop/library/Newscoop/Services/Plugins/ManagerService.php

```diff
@@ -98,7 +109,7 @@ public function installPlugin($pluginName, $version, $output, $notify = true)
             throw new \Exception("Plugin name is invalid, try \"vendor/plugin-name\"", 1);
         }
 
-        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar require --no-update ' . $pluginName .':' . $version .' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar update ' . $pluginName .'  --prefer-dist --no-dev -n');
+        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.qphar require ' . $pluginName .':' . $version);
```

Comment:
I guessed that it probably had something to do with an old version of
composer.


On Tue, Mar 22, 2016 at 10:50 AM, Paweł Mikołajczuk <
notifications@github.com> wrote:

> In newscoop/library/Newscoop/Services/Plugins/ManagerService.php
> <https://github.com/sourcefabric/Newscoop/pull/1347#discussion_r56959470>:
>
> > @@ -98,7 +109,7 @@ public function installPlugin($pluginName, $version, $output, $notify = true)
> >              throw new \Exception("Plugin name is invalid, try \"vendor/plugin-name\"", 1);
> >          }
> >
> > -        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar require --no-update ' . $pluginName .':' . $version .' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar update ' . $pluginName .'  --prefer-dist --no-dev -n');
> > +        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.qphar require ' . $pluginName .':' . $version);
>
> i wrote it before as composer had weird behaviour at that time with
> require and only fix was to call update after that.
> [image: :+1:] from me.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sourcefabric/Newscoop/pull/1347/files/092aafa5c9f683790542be69c37d806eafec3544#r56959470>
>